.circleci/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+jobs:
+  build:
+    macos:
+      xcode: "8.3.3"
```

Comment:
Any reason to use this version rather than update to "9.4.0"

Reply:
Good call. Updated. We'll see if it passes